.github/workflows/ci.yml

```
@@ -73,22 +73,40 @@ jobs:
             })
             console.log('Deleted comment with ID ${{ steps.test-results-comment.outputs.comment-id }}')
 
-  nuxt-playwright:
-    name: Nuxt Playwright tests
+  nuxt-playwright-e2e:
+    name: Nuxt Playwright e2e tests
     runs-on: ubuntu-latest
 
     steps:
       - uses: actions/checkout@v3
 
       - uses: ./.github/actions/setup-node-env
 
-      - name: Run Nuxt Playwright tests
-        run: pnpm test:playwright --workers=2
+      - name: Run Nuxt Playwright e2e tests
+        run: pnpm test:playwright e2e --workers=2
 
       - uses: actions/upload-artifact@v3
         if: failure()
         with:
-          name: nuxt-test-results
+          name: nuxt-e2e-test-results
+          path: test-results-e2e
```

This should still be `test-results/` unless we change the `outputDir` in the Playwright config.

```suggestion
          path: test-results
```